GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java

```diff
@@ -2679,4 +2685,19 @@ public Object getTag() {
         return mTag;
     }
 
+    private final HandlerThread mHandlerThread;
+    private final Handler mHandler;
+
+    public void runOnTheFrameworkThread(final Runnable runnable) {
```

Comment:
Maybe. While working on this patch I did not identify requirements that demand that.